packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -299,6 +304,17 @@ export class SchemaConverter implements SimpleDbSchemaConverter {
           });
       });
   }
+
+  private rewriteCanonicalIds(
+    txn: SimpleDbTransaction
+  ): PersistencePromise<void> {
+    const targetStore = txn.store<DbTargetKey, DbTarget>(DbTarget.store);
+    return targetStore.iterate((key, value) => {
```

Comment:
The variable names are somewhat confusing. Consider renaming:

  * `value` => `originalDbTarget`
  * `originalTargetData` leave alone
  * `updatedTargetData ` => `updatedDbTarget`

Reply:
Good improvement. Done.